bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4727,5 +4727,35 @@
         </sql>
     </changeSet>
 
+    <changeSet id="create-teleconsultation-link-role-202212012000" author="Kushboo">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM role WHERE role = 'Teleconsultation:CreateLink';
+            </sqlCheck>
+        </preConditions>
+        <comment>Creating new role Teleconsultation:CreateLink for Teleconsultation module</comment>
+        <sql>
+            INSERT INTO role(role, description, uuid) VALUES ("Teleconsultation:CreateLink", "Ability to create teleconsultation link in teleconsultation module", uuid());
```

Comment:
If someone has the ability to create an Appointment, then I think they can create a TeleCon appointment too. Should not need a new role. This privilege can be added in createAppointment Role itself.

I am not sure what is the need for a new role?

Reply:
Updated, now appointments role has create teleconsultation privilege